### DISCUSSION TITLE
Considering some testnet content creators as Founding Members

### DISCUSSION THREAD

**Discussion by eclipsingbinary** [Joystream/joystream#3392] [General]
Title: A cathedral without the art is just a building.

"They should have sent a poet". 
This is a famous quote by the character Ellie Arroway in Carl Sagan's book Contact. Her technical knowledge as a scientist paid no justice to what she was seeing when she travelled to another universe as she could not find the words to describe it.

This document will hopefully make the case that JoystreamDAO should consider some early adopter content creators as founding members of the DAO. Not just any content creator but those that are recognized by the community as active, invested in and helpful in bringing the project forward with their content.

1. It's not about JoystreamDAO

This argument proposed in this document is not only about JoystreamDAO but it is a much broader scope, understanding that these are changing times in the way we interact and transact on the internet, but most importantly, understanding the shift in mindset required by culture to support these new ideas and technologies. It is my estimation (not my idea) that this is and has always been best done by ART. "Art is the lie that tells the truth" (quote attributed to many famous artists in history). 
A science fiction writer (emphasis on the fiction) will inspire way more people about the future than any technical document. Before there was science there was philosophy and before philosophy there was already art. It's no coincidence as art gets to the deepest of our subconscious.
What I mean is that it's about bringing about culture. So it is not about JoystreamDAO, it's about all DAOs. We need Art and Poetry and Music to bring about the future in an inspiring way.

2. Not all content creators are the same.

More of a continuation on the first point. Not all content creators are well versed in tech and culture. Most content creators, at the point in which this document is written, are still very much in Web2, and a big chunk of those will never go out into web3 but will instead wait for the platform in which they are already in to accept crypto payments and will happily accept their fate inside those walled gardens and will never skip a beat when someone brings up decentralization to them, especially if it's explained to them by technical people. For this you need a special breed of content creators, the early adopters that risked dropping Web2 in order to not just bring about Web3 but also to put it to the test. The ones that know that it is cultural, political and economic as much as it is technological. Those that push through the non user friendly interfaces to make sure that new platforms have users to test their tech with. Napster, Limewire or Kazaa would have not been P2P networks without the users that ventured out there in the undergrounds of culture to bring those into the mainstream. You need those who are daring and willing to disrupt.

3. For Team Human

"The special thing that we, regular biological humans have that the robots don't, is soul". -Douglas Rushkoff

Technocratic systems have always lacked humanity. We need a cultural, philosophical and ethical underpinning to the systems that we build. This is best done by being wrapped in story (art). This is why Shakespeare's plays are still to this day great representations of human behaviour, more so than technical psychology books. In them, we find the nature of relationships, self deprication, abuse of power, envy, greed, love and much more. 
This is why I've titled this document, A cathedral without the art is just a building. We need the storytelling as human beings to move us, to hack awe, to compell us. We can't escape the storytelling even if we wanted to as we live inside narratives. These are best constructed by artists that understand culture. It gets to the human level.
We are moving to a highly technical not too distant future and the fear of many is that it will lack humanity. We need, for lack of a better word and to copy Rushkoff, "Soul". What I mean is that, us, the HUMANS that are using these new systems should have a decentralized ability to govern the new systems that are being built, if only to put in check the highly technical people that build them to avoid  the abuses that rob us from our humanity i.e Web2 and weaponization of data. 

4. You can't test a video platform without videos
This is a straight forward point and is strictly about JoystreamDAO.
You don't just want early adopters during testnet period you absolutely more than anything NEED them as you can't put a video platform to the test unless you have a large amount of videos, and it is only fair that community members that are uploading videos on the regular during testnet be considered as founding members as they are the foundation in which the whole platform is able to be tested and deployed. 
They should be awarded DAO/FM tokens and should have voting power relative to their participation during this instrumental time. Things to consider may be time on Atlas, number of videos, participation in video bounties, referrals, and if they are active in the community. This determines the amount of tokens awarded and therefore the voting power that each content creator has. 

Conclusion:
It is my hope that this non-technical short document can illuminate to the Founding Members of JoystreamDAO that content creators, especially artists, are not only helpful but necessary in the development and communication of the project and it's goals to the broader population. 

Let us, the content creators, tell our decentralized version of the story to those that follow us in order to help grow the JoystreamDao community and consider that a valuable contribution.

Thank you for your consideration and for reading

Sincerely,
Eclipsing Binary

**Comment by bedeho** [comment]
I appreciate the input, and suppose I buy your points, I think what I am still missing is concrete proposition which actually would move us in the direction you are talking about. So who, gets what and how much, for doing what, as judged by whom. In particular, the scheme has to really explain how to exactly capture the essence of your core claims, for example that not all creators are the same. You would need to have a practical scheme for rewarding these daring creators, and not anyone looking to make a quick buck. Our lack of a good creator program is not out of stinginess, its that we don't have the time or wisdom to make it practical to generate satisfactory answers to such questions at this time.

**Reply by eclipsingbinary** [reply]
I appreciate that, those questions I don't have the answer to. I am new here. I am however participating and learning from TEC (Token Engineering Commons) and may be to offer a more concrete proposition at a later time. My point when I say that not all creators are the same is precisely what differentiates those that want to make a quick buck and those of us that actually care about what is going on inside the projects, and the community (Discord) should be able to tell the difference. But you are right. I don't know the how to. I will continue to think it over and come back with something more concrete. thank you for reading and engaging

**Reply by bedeho** [reply]
How about we just ideate a bit on it here?

So I think that, before we can define how to reward quality creators, we have to just define what sort of characteristics we are looking for. You mention

>  For this you need a special breed of content creators, the early adopters that risked dropping Web2 in order to not just bring about Web3 but also to put it to the test. The ones that know that it is cultural, political and economic as much as it is technological. 

Perhaps you can add more detail here about these types of creators and their content, for example:
- where they would publish
- on what topics
- what sort of content types they publish: (commentary, bio, vlog, debates, monologues, etc.)
- their motivation
- how they build their audience
- how they think about monetization

and anything else you can think of, the more structured the better. I am sure you can divide these people into multiple categories perhaps, where there would be sharper answers for these questions, so feel free to do that also.

Once we have this, we will have some plausible personas to build on, and it becomes much easier for us to think of effective reward schemes, and also ways to actually reach these people.

Really appreciate any help you can provide here :)

**Reply by eclipsingbinary** [reply]
Thank you @bedeho I'll think more deeply about the points you bring up and try to come up with some ideas and more definite answers for those. I admit that I first have to do a lot more reading of the github and the forum so that it makes sense. 
Thank you sir 🙏

**Reply by bedeho** [reply]
Very much looking forward to it, this can have a _big_ impact for us!

**Comment by mochet** [comment]
Really well written and compelling discussion you've started. You make very interesting and valid points.
I definitely want to see at least some content creators become Founding Members, I'll definitely think about it and see if there's some way to help address it (I don't make decisions though, so the end choice isn't up to me)
We will be having `bounties` that can contribute to a user becoming a Founding Member and I very much hope to see some that can be appealing or inspiring for you.
Besides that, based upon what you've written here I definitely think you'd easily be able to participate in some platform roles that exist within the FM program--they may from the outside appear to require extreme amounts of knowledge but they are doable and myself and others will definitely be there to help.


Outside of the Founding Member program...
"identifying and rewarding art" happens to be a topic I'm obsessively interested in! So I'm glad to see someone else talking about it with passion.

It's an area that requires a lot of thought for this platform, due to being `fully decentralized` a lot of the typical tools that could address this easily don't really work. Features would have to exist within the `runtime code` that make technical and economic sense--which isn't an easy puzzle to solve, because you can't really stop a `bad actor` from participating. You *could* designate `judges` to reward art, but that is its own topic (:

Valuing content creators is something I've really thought about extensively and I've explored a lot of ideas. I hope at least some of them end up being feasible to develop after the platform launches. These are some of the considerations I've had, which might help to highlight some of the problems that will likely be encountered along the way:
* Rewarding content is much the same as surfacing content, what content is shown to users matters.
* I'm very mindful of ensuring "art" and "advertising" are separated. This is a very difficult distinction and any system that allows for voting on any capacity will be abused by advertisers.
* I think view counts are a very limited way to define the "value" of art.
* I don't really like the idea of a single `judge` having sole authority to value content.
	* I've experimented with "social systems" on our testnet that require more than one judge. We have two programs for either "public domain" works or "original" works that reward users and although the idea of having 3 judges for each submission is viable and can provide a weighted score, it just requires a lot of work because you need more people dedicated to doing it.
		* A lot of the effort I've put in has really been focused on pushing governance forward because its the area that is the hardest to gain interested users and its the most fundamental layer of this platform. Content creators are obviously critical, but the governance system has to work reliably for anything else to work. In early stages it was difficult to even get reports written on a weekly basis, or have enough elected council members vote on proposals at all.
* I've thought a lot about voting systems for content, specifically in the realm of categorizing content and token curated registries. I haven't seen a system that explores this so far.
	* I think its much easier for a community to vote/value a category of content rather that individual creators or channels. In this day and age, new categories for every type of content are invented every minute so this is a really interesting space.

**Reply by eclipsingbinary** [reply]
Thank you guys for continuing this thing. I'm not trying to grab money in anyway, I have Defi for that. This is more about including the people, especially the early adopters. 

I will join some of the bounties for sure at some point soon, and I have and will continue to refer people to JS.

Sorry I'm not technical so my writings aren't with specific points in mind and are more general. Someone else like you guys have may have a way to organize things better.

In TEC's Discord (Token Engineering Commons) they have the concept of !praise

People in the community that have the power to give !praise  can do so to anyone in the Discord. It can only be given about something specific though. example: !praise @name for having created this piece of art and sharing it and tagging us in their social media. Or !praise @name for having found an important bug and reported it here. 

What's cool about !praise is that it can and does turn into TEC tokens because the community as a whole found value in what was done.

I know that they have helped other DAOs adopt stuff like this and design Discord bots for them to help them with their goals and onboarding. Maybe something to look into.

https://tecommons.org

**Reply by mochet** [reply]
> Thank you guys for continuing this thing. I'm not trying to grab money in anyway, I have Defi for that. This is more about including the people, especially the early adopters.
> 
> I will join some of the bounties for sure at some point soon, and I have and will continue to refer people to JS.
> 
> Sorry I'm not technical so my writings aren't with specific points in mind and are more general. Someone else like you guys have may have a way to organize things better.

(I apologize as this is a little long, but I hope it can help to inspire you and show you the value you have--I want to show you there really is a spot for non-technical people)

Really, do not worry at all about not being technical. I think there's a significant quality in what you're writing about and the passion you've demonstrated here that you maybe don't realize. You're basically very close to participating in governance. Not only can you very likely easily participate in the FM program roles (I still want content creators to be able to join somehow) but the kind of impact you can have just by writing and exploring ideas can really give you an outsized impact on things.

I very much have a similar non-technical perspective, I have a background in creative stuff and I've used computers for a long time, but I just wasn't born on the `coder` side of things. While I know a lot about how this platform works, it wasn't through programming or being a coder, it was by writing, reading and screwing around with ideas and by asking naive questions or things that I thought were stupid.

As you wrote in your original post:
	`What I mean is that, us, the HUMANS that are using these new systems should have a decentralized ability to govern the new systems that are being built`
This is exactly what you're already doing when you start discussing ideas and concepts for bots that could be helpful. I've been asked a few times who I think will be the most valuable participants for our community to have, and I can't remember saying anything besides writers.

One of the other things you wrote:
	`This determines the amount of tokens awarded and therefore the voting power that each content creator has.`
Tokens do allow you to vote and do give you voting power but in reality *anyone* can go and buy tokens and get voting power. If you understand how governance works here and if you can write in a compelling way--then you can actually meaningfully introduce your own ideas and shape and guide the future. Being able to do that allows you to *actually* change things in the ways you care about and be far more than a voter.

You clearly have a passion for art and valuing content creators, you can write about it well and you care about it, and if you learn where to put that energy, you can really have an impact. Your passion is something that isn't seen very often and really reminds me a lot about the passion I had when I first came across this project--much like you I knew I really wanted to be a part of things, I just didn't know how to. It took me a while to realize how to best direct that passion and I didn't get there alone, the people at JSG (early on there really wasn't any community) and people that came together within our community and helped shape where to put my passion.

You have a huge benefit from having something no one that is already in our community has, which is a fresh pair of eyes. It means your perspectives are new and are very valuable. Our community is quite small compared to a lot of other communities, and this project is overwhelming due to complexity. But a small community means that a person with passion, that is directed can have a huge impact.
I had to learn and build things myself along the way because nothing existed, but over time a little bit of structure has sort of formed by itself and I am more than happy to help teach you how to make sense of it.

It turns out that in the crypto world a lot more people just want to buy tokens compared to those willing to participate--voting is a valuable form of participation but there are projects out there where a lot of people don't even know how to vote or just aren't interested in voting. There is definitely space here for passionate people and fresh voices.

One of the things I learned early on and have been able to appreciate a lot more over time is that even though this web3 stuff is very sophisticated and while coding is a very valuable skill, if you don't have the knack for it, it doesn't matter because you can create your own path and role in things. There's just as much value that non-technical people provide. 

Some governance decisions are technical, but many are just writing ideas that are possibly feasible. Just as you described a bot here, I want to show you how to take something like that and how easy it can be to make it happen.
I wrote a proposal to create a bot a while ago. I don't know a thing about how bots are coded or what is required, but its easy to ask members of our community who do know those things.
This is how simple it can be and it'll be even easier quite soon:
1. https://pioneer.joystreamstats.live/#/proposals/17 - Wrote a brief description for a Discord bot that could post notifications of each new video uploaded to Joystream. I didn't write technical stuff, I just described an idea I thought would be valuable.
2. https://testnet.joystream.org/#/forum/threads/362?replyIdx=7 - Someone expressed interest in completing the bot
3. https://github.com/Joystream/community-repo/pull/196 - They coded the bot. I don't know technical stuff, so someone else reviewed the code and they fixed a thing or two.
4. https://testnet.joystream.org/#/proposals/154 - I made a proposal to pay the user who coded it and they were paid (from the council's budget, I didn't pay for it!)
5. There's now a channel on our Discord called `#video-bot` and there's now a bot that does what I described (it's currently down, but it usually works very well!)

Exploring, creating and even just being a part of the discussion when it comes to ideas and concepts is really valuable. While writing in a way that coders can understand is a skill that takes a little time to understand (I am still learning this) the world be very boring without people writing and sharing ideas that then get built, the same as the title you used for your writing.
We have people in our community that can not only help to discuss and inform how viable an idea is when it comes to technical stuff, they can code it.

Stuff like this Discord bot you've mentioned is valuable stuff to think and write about. GitHub is really where the most technical stuff happens on this project and it can be quite intimidating to understand how it all fits together--I would highly recommend taking a look at `issues` that have been written in our "community-repo": https://github.com/Joystream/community-repo/issues?q=is%3Aissue. The community-repo is where our community members submit work and discuss ideas (partially there as well as in other places).
You'll find issues discussing things you're thinking about there--bots, rewards, whatever, they all start from someone writing something. This repo (which is https://github.com/Joystream/joystream) unlike that one is really moving at a frantic pace given how close we are getting to launch and while you can still start discussions here it may be difficult to get replies.

I want to share with you some examples of just simple writing I've done that has somehow magically turned into something--I haven't written a line of code, but just writing and discussing ideas is really the fundamental way of how things get started. I had no idea that anything I wrote would ever matter in this space because I'm non-technical. It took me a long time to find my footing but sharing a simple idea and talking about it is what ends up in it actually turning into something. Writing something and seeing people come together to help develop it further and maybe build it is an amazing experience.

The first time this happened for me was quite a while ago https://github.com/Joystream/community-repo/issues/50. I wrote an idea consisting of 2 paragraphs. Just a week later a user, who has become a really amazing member of our community, turned up and turned those 2 paragraphs into an entire statistics website built just for this platform. It's been significantly improved upon since then.

I wrote this: https://github.com/Joystream/joystream/issues/1157 - Somehow that idea made sense. It's been more than a year since it was accepted and I've only just been able to use the function for the first time very recently with my own two hands.

I wrote this https://github.com/Joystream/joystream/issues/2595 - Somehow it made sense, got a really good reaction and was implemented within the platform's code.

I didn't write these because I'm an expert coder, I wrote them because I'm far from being a coder and was inspired by things I had seen in the world, my own personal experiences or out of simple frustrated about things that didn't exist. I just wrote about ideas that sounded like they might make sense. I think its more than obvious we will need contributors on this project who have outside perspectives, have understanding of content creators and are passionate about it. Someone has to write something to start discussions that define the roles and expectations in everything that happens.

When the upgrade is launched our new forum will be a huge part of things, and I really hope you contribute there. I'll be putting a lot of energy into it, I definitely want us to all build an amazing, interesting forum and that requires writing.
Just the simple act of posting on the forums is particularly important--every single one of them contributes to your reputation on the platform. It's not like Discord where messages disappear after a few hours, the forum acts as a "map" of this project and its structure heavily impacts things. Every single proposal also contributes to your reputation.

I totally get that this project is quite complex and a little overwhelming at first just because of how many systems there are. No matter how great systems are on this or any other project, someone will always be needed to write, make passionate arguments and be a part of defining the future they want to see and that they want to be a part of.

I also really want to emphasize the point about "reputation". If you have a good reputation, that is rich in history, then it can significantly impact things once we launch. Funding is available through proposals and bounties that are well written. This DAO can "hire" people as "workers" for roles that haven't been invented yet, you could technically write your own job description and present it as a proposal. There's really going to be a spectacular world to explore and someone needs to be the person defining what it can be and then will be through writing.

> In TEC's Discord (Token Engineering Commons) they have the concept of !praise
> 
> People in the community that have the power to give !praise can do so to anyone in the Discord. It can only be given about something specific though. example: !praise @name for having created this piece of art and sharing it and tagging us in their social media. Or !praise @name for having found an important bug and reported it here.
> 
> What's cool about !praise is that it can and does turn into TEC tokens because the community as a whole found value in what was done.

That is really interesting. Have a think about who should have access to it, maybe it could be voted via a proposal? Also how might it work with the platform's features (like forums or videos)?

> I know that they have helped other DAOs adopt stuff like this and design Discord bots for them to help them with their goals and onboarding. Maybe something to look into.
> 
> https://tecommons.org

I will take a look at this, it sounds like it'll be fun to read about.

**Reply by eclipsingbinary** [reply]
Wow, thank you so much @mochet for all the detailed information, all the references to look at and for your uplifting words that really mean a lot to me. I will go through these links to try to understand more deeply and shift my focus to the forum to try to write better and in a way that makes more sense for the project. Thank you friend 🙏

**Comment by MindsGaming** [comment]
Title: I'm just a gamer

"I think the key divide between the interactive media and the narrative media is the difficulty in opening up an empathic pathway between the gamer and the character, as differentiated from the audience and the characters in a movie or a television show."
~ Steven Spielberg

Why Make Early Adoptiors Founders?

1.) Burnout 
Many beta and alpha users will suffer from a symptom called burnout, this is simply giving a lot of attention at the very start of exploring a product but ending with no content and very limited interaction with the product.

2.) Community & Culture
The first active users on a program are the ones that usually create and form the culture and community of a platform.

3.) Content 
Why wouldn't you want to provide content to platform where a founder and help shape the culture and community.

4.)Growth
Earlier adapters of the platform are the ones who grow the platform, to thank them for growing and helping the platform expand into a active community.

Life is a game

The ability for creators to not only be independent but thrive in their own decentralized community with growth and support back from the community they are providing content in an inspired and dutiful way isn't just part of web3 but the reason to support web3. 

I think the real question is how to accomplish making early adaptors into founders in a fair way ; not if it should be done.

**Comment by MindsGaming** [comment]
https://github.com/Joystream/joystream/issues/3519
